go/stackman/api/v1/cliquel1_types.go

```diff
@@ -0,0 +1,939 @@
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000dc": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000dd": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000de": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000df": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e0": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e1": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e2": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e3": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e4": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e5": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e6": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e7": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e8": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000e9": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ea": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000eb": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ec": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ed": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ee": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ef": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f0": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f1": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f2": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f3": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f4": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f5": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f6": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f7": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f8": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000f9": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fa": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fb": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fc": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fd": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000fe": {
+      "balance": "0x1"
+    },
+    "00000000000000000000000000000000000000ff": {
```

Comment:
Why are there balances going all the way to `0xff`?

Reply:
Puppeth adds these - they're prefunded precompile addresses, see https://ethereum.stackexchange.com/questions/68056/puppeth-precompile-addresses.